addon/components/ember-table.js

```diff
@@ -54,7 +59,17 @@ export default class EmberTable2 extends Component {
    * does nothing), 'single' (clicking on a row selects it and deselects other rows), and 'multiple'
    * (multiple rows can be selected through ctrl/cmd-click or shift-click).
    */
-  @property selectionMode = 'single';
+  @property selectionMode = SELECTION_MODE_SINGLE;
+
+  /**
+   * A configuration that controls how columns shrink (or extend) when total column width does not
+   * match table width. Behavior of column modification is as follow:
+   * 1) "none": nothing changed to the column
+   * 2) "equal_column": extra space is distributed equally among all columns
+   * 3) "first_column": extra space is added into the first column.
+   * 4) "last_column": extra space is added into the last column.
+   */
+  @property tableFillupMode = TABLE_FILLUP_MODE_NONE;
```

Comment:
Let's call this `tableResizeMode`, without context I don't think people will understand what "fillup" means here, whereas "resize" gives them a bit more of a hint